src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortSamSparkIntegrationTest.java

```diff
@@ -20,9 +23,8 @@
                 {"count_reads.cram", "count_reads_sorted.cram", "count_reads.fasta", ".cram", "coordinate"},
                 {"count_reads.bam", "count_reads_sorted.bam", "count_reads.fasta", ".cram", "coordinate"},
 
-                //SortBamSpark is missing SORT_ORDER parameter  https://github.com/broadinstitute/gatk/issues/1260
-//                {"count_reads.bam", "count_reads.bam", null, ".bam", "queryname"},
-//                {"count_reads.cram", "count_reads.cram", "count_reads.fasta", ".cram", "queryname"},
+                {"count_reads.bam", "count_reads.bam", null, ".bam", "queryname"},
+                {"count_reads.cram", "count_reads.cram", "count_reads.fasta", ".cram", "queryname"},
```

Comment:
Is this a valid test? The inputs and outputs are the same! I'm not convinced that queryname sorting is actually working in this branch, since you use a `ReadCoordinateComparator` in the tool no matter what.